src/Users/README.md

```diff
@@ -123,6 +180,82 @@ if (!$result) {
 }
 ```
 
+### DELETE OWN ACCOUNT
+```php
+$result = \ATDev\RocketChat\Users\User::deleteOwnAccount(hash("sha256","[USER PASSWORD]"));
+if (!$result) {
+    $error = \ATDev\RocketChat\Users\User::getError();
+}
+```
+
+### DEACTIVATE IDLE
+```php
+$count = \ATDev\RocketChat\Users\User::deactivateIdle(5, "guest");
+if (!$count) {
+    $error = \ATDev\RocketChat\Users\User::getError();
+}
+```
+
+### PRESENCE
+```php
+$presence = \ATDev\RocketChat\Users\User::presence();
```

Comment:
As far I understand this gets if the current user in on chat. Thus we have to change this to non0static call.